controller-server/src/main/java/com/yahoo/vespa/hosted/controller/LockedApplication.java

```diff
@@ -195,77 +196,77 @@ public LockedApplication withoutDeploymentJob(JobType jobType) {
-                                     metrics, pemDeployKey, rotation, rotationStatus);
+                                     metrics, pemDeployKey, rotations, rotationStatus);
     }
 
     public LockedApplication withOwnershipIssueId(IssueId issueId) {
         return new LockedApplication(lock, id, createdAt, deploymentSpec, validationOverrides, deployments,
                                      deploymentJobs, change, outstandingChange, Optional.ofNullable(issueId), owner,
-                                     majorVersion, metrics, pemDeployKey, rotation, rotationStatus);
+                                     majorVersion, metrics, pemDeployKey, rotations, rotationStatus);
     }
 
     public LockedApplication withOwner(User owner) {
         return new LockedApplication(lock, id, createdAt, deploymentSpec, validationOverrides, deployments,
                                      deploymentJobs, change, outstandingChange, ownershipIssueId,
                                      Optional.ofNullable(owner), majorVersion, metrics, pemDeployKey,
-                                     rotation, rotationStatus);
+                                     rotations, rotationStatus);
     }
 
     /** Set a major version for this, or set to null to remove any major version override */
     public LockedApplication withMajorVersion(Integer majorVersion) {
         return new LockedApplication(lock, id, createdAt, deploymentSpec, validationOverrides, deployments,
                                      deploymentJobs, change, outstandingChange, ownershipIssueId, owner,
                                      majorVersion == null ? OptionalInt.empty() : OptionalInt.of(majorVersion),
-                                     metrics, pemDeployKey, rotation, rotationStatus);
+                                     metrics, pemDeployKey, rotations, rotationStatus);
     }
 
     public LockedApplication with(MetricsService.ApplicationMetrics metrics) {
         return new LockedApplication(lock, id, createdAt, deploymentSpec, validationOverrides, deployments,
                                      deploymentJobs, change, outstandingChange, ownershipIssueId, owner, majorVersion,
-                                     metrics, pemDeployKey, rotation, rotationStatus);
+                                     metrics, pemDeployKey, rotations, rotationStatus);
     }
 
     public LockedApplication withPemDeployKey(String pemDeployKey) {
         return new LockedApplication(lock, id, createdAt, deploymentSpec, validationOverrides, deployments,
                                      deploymentJobs, change, outstandingChange, ownershipIssueId, owner, majorVersion,
-                                     metrics, Optional.ofNullable(pemDeployKey), rotation, rotationStatus);
+                                     metrics, Optional.ofNullable(pemDeployKey), rotations, rotationStatus);
     }
 
     public LockedApplication with(RotationId rotation) {
```

Comment:
This should be changed to `with(List<RotationId>)` as it's currently unclear whether this means append or replace.

Reply:
Leaving this as-is for now. Will change this in a different PR where multiple rotations are assigned to an applications.